lib/modules/apostrophe-schemas/index.js

```diff
@@ -1619,6 +1619,27 @@ module.exports = {
       }
     });
 
+    self.addFieldType({
+      name: 'email',
+      converters: {
+        string: function (req, data, name, object, field, callback) {
+
+          if (field.required && (_.isUndefined(data[name]) || !data[name].toString().length)) {
```

Comment:
Use `self.apos.launder.string()` to definitely convert it to a string, without missing funny cases like  `null` (which has no `toString` method and will crash). Then you can do your length test.